includes/Core/Modules/Modules.php

```diff
@@ -391,6 +408,27 @@ public function get_module_sharing_settings() {
 		return $this->sharing_settings;
 	}
 
+	/**
+	 * Sorts modules by their order property.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param  array $modules Array of Module objects.
+	 * @return array Sorted array of Module objects, maintaining the original array's keys.
+	 */
+	public function sort_modules( $modules = array() ) {
+		uasort(
+			$modules,
+			function( Module $a, Module $b ) {
+				if ( $a->order === $b->order ) {
+					return 0;
+				}
+				return ( $a->order < $b->order ) ? -1 : 1;
+			}
+		);
+		return $modules;
+	}
```

Comment:
Now that this is only used in one place inside the class, let's either remove this method or make it private, since it shouldn't be necessary to call it from the outside.